apis/userapi/userapi.yaml

```diff
@@ -1820,7 +1932,7 @@ definitions:
     properties:
       email:
         type: string
-        description: 'Represents the email of the user'
+        description: 'The email of the user'
       firstName:
```

Comment:
@anmol2302 Is it Ok to add descriptions for firstName and lastName fields?
On line 1956, can you clarify the description for phoneVerified field? the description talks of 'provider'. there is no such field.
On line 1958, the field 'subject' has no description and is an array. could you clarify the same?